src/components/blog/post/body/_image.scss

```diff
@@ -1,15 +1,18 @@
 img {
+  margin: 40px 0;
+
+  @include media(">=desktop") {
+    margin: 56px 0;
+  }
+}
+
+img:not(:global .gatsby-resp-image-image) {
```

Comment:
The attributes here were messing with the plugin's own styles, which introduces css classes, and some html structure to render a low-res preview and center the image.

So I changed this css to only use these positioning rules for images not rendered with the plugin. The margin part, I kept above because it seems to make sense for both

Reply:
Thank you for the explanation. I was getting a bit triggered to be honest, mostly because I spent so much time aligning all of this. :sweat_smile:

Doesn't look good but I guess there's not much we can do since the plugin forces this on us. :/